securing_apps/topics/overview/supported-platforms.adoc

```diff
@@ -53,7 +53,10 @@ endif::[]
 * <<_saml_jboss_adapter,JBoss EAP>>
 ifeval::[{project_community}==true]
 * <<_saml_jboss_adapter,WildFly>>
-* <<_tomcat_adapter,Tomcat>>
+* <<_saml-tomcat-adapter,Tomcat>>
+endif::[]
+* <<java-servlet-filter-adapter,Servlet filter>>
```

Comment:
Could the name of the anchor follow the same standard as other SAML adapters, e.g. `_saml-java-servlet-filter-adapter`?

Reply:
Okay, that works.  Updated.